torch/_inductor/config.py

```diff
@@ -201,6 +201,8 @@ class cpp:
     # enable weight prepacking to get a better performance; may lead to large memory footprint
     weight_prepack = True
 
+    max_horizontal_fusion_size = max_fusion_size
```

Comment:
What is the point of this is it is the same as `max_fusion_size`?  Won't `max_fusion_size` already prevent the fusion?

Reply:
Currently, the `max_horizontal_fusion_size` is a heuristic value, and I suppose the default value of `max_fusion_size` has been proven to be good for performance. Hence, I set the default value to `max_fusion_size`. I will change it to the magic value.

Reply:
> default value of max_fusion_size has been proven to be good for performance

`max_fusion_size` is completely untuned and is just a random guess.  


What types of speedups are you seeing from a different value here?